### ISSUE TITLE
Saving an new outgoing WebHook integration causes server error

### ISSUE THREAD

**Issue by mischah** [RocketChat/Rocket.Chat#3293]
Rocket.Chat version: 0.3.1

I’m trying to setup the Giphy integration as described [here](https://github.com/FinndropStudios/GifRocket).

After filling all mandatory fields and hitting the »save changes« button a server button occurs.

Log:
```
I20160518-22:24:32.234(2)? Exception while invoking method 'addOutgoingIntegration' TypeError: Cannot call method 'split' of undefined
I20160518-22:24:32.234(2)?     at [object Object].Meteor.methods.addOutgoingIntegration (packages/rocketchat_integrations/server/methods/outgoing/addOutgoingIntegration.coffee:26:40)
I20160518-22:24:32.234(2)?     at [object Object].methodsMap.(anonymous function) (server/lib/debug.js:23:26)
I20160518-22:24:32.234(2)?     at maybeAuditArgumentChecks (livedata_server.js:1698:12)
I20160518-22:24:32.234(2)?     at livedata_server.js:708:19
I20160518-22:24:32.234(2)?     at [object Object]._.extend.withValue (packages/meteor/dynamics_nodejs.js:56:1)
I20160518-22:24:32.235(2)?     at livedata_server.js:706:40
I20160518-22:24:32.235(2)?     at [object Object]._.extend.withValue (packages/meteor/dynamics_nodejs.js:56:1)
I20160518-22:24:32.235(2)?     at livedata_server.js:704:46
I20160518-22:24:32.235(2)?     at tryCallTwo (/Users/mkuehnel/.meteor/packages/promise/.0.5.1.8idxpg++os+web.browser+web.cordova/npm/node_modules/meteor-promise/node_modules/promise/lib/core.js:45:5)
I20160518-22:24:32.235(2)?     at doResolve (/Users/mkuehnel/.meteor/packages/promise/.0.5.1.8idxpg++os+web.browser+web.cordova/npm/node_modules/meteor-promise/node_modules/promise/lib/core.js:171:13)
```

**Comment by mischah** [comment]
Okay, according to [addOutgoingIntegration.coffee:26](https://github.com/RocketChat/Rocket.Chat/blob/develop/packages/rocketchat-integrations/server/methods/outgoing/addOutgoingIntegration.coffee#L26) entering a channel isn’t optional. At least in this line of code. 

Going to dig deeper now.

**Comment by graywolf336** [comment]
Closing, this was fixed via #3275

**Comment by mischah** [comment]
👌
